src/components/utility/Typography/Typography.module.css

```diff
@@ -17,13 +17,13 @@ h3 {
 
 .h3,
 .body1 {
-  font-size: 0.9rem;
+  font-size: 1.125rem;
   line-height: 1.75rem;
 }
 
 .body2 {
-  font-size: 0.9rem;
-  line-height: 1.5rem;
+  font-size: 0.9375rem;
+  line-height: 1.563rem;
```

Comment:
Not trying to be nit picky here but it makes it easier to figure out the source of spacing discrepancies.